app/src/main/java/org/oppia/app/home/HomeActivityPresenter.kt

```diff
@@ -21,26 +21,26 @@ class HomeActivityPresenter @Inject constructor(private val activity: AppCompatA
     setUpNavigationDrawer()
     if (getHomeFragment() == null) {
       activity.supportFragmentManager.beginTransaction().add(
-        R.id.home_fragment_placeholder,
+        R.id.home_activity_home_fragment_placeholder,
         HomeFragment(),
         TAG_HOME_FRAGMENT
       ).commitNow()
     }
   }
 
   private fun setUpNavigationDrawer() {
-    val toolbar = activity.findViewById<View>(R.id.home_activity_toolbar) as Toolbar
+    val toolbar = activity.findViewById<View>(R.id.home_activity_home_toolbar_toolbar) as Toolbar
     activity.setSupportActionBar(toolbar)
     activity.supportActionBar!!.setDisplayShowHomeEnabled(true)
     navigationDrawerFragment =
-      activity.supportFragmentManager.findFragmentById(R.id.home_activity_fragment_navigation_drawer) as NavigationDrawerFragment
+      activity.supportFragmentManager.findFragmentById(R.id.home_activity_drawer_fragment_navigation_drawer) as NavigationDrawerFragment
     navigationDrawerFragment!!.setUpDrawer(
-      activity.findViewById<View>(R.id.home_activity_drawer_layout) as DrawerLayout,
+      activity.findViewById<View>(R.id.home_activity_home_drawer_drawer_layout) as DrawerLayout,
```

Comment:
Similarly here `home_activity_home_drawer_layout`

Reply:
`<parent_file_name>_<view_name>_<widget_type>`, this may be `home_activity_home_drawer_drawer_layout`